### PR TITLE
Fix for CNW tests

### PR DESCRIPTION
**Pull request by chessky**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java [hunk 1]
```diff
@@ -25,8 +25,8 @@ public void createNewWikiLangTC001(String lang) {
     WikiBasePageObject base = new WikiBasePageObject();
     CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
     cnw1.selectLanguage(lang);
-    String expectedDomainPrefix = "en".equals(lang) ? "" : String.format("%s.", lang);
-    assertEquals(cnw1.getDomainPrefix().getText(), expectedDomainPrefix);
+    String expectedDomainSufix = "en".equals(lang) ? "" : String.format("%s", lang);;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java]: MAJOR Remove this empty statement. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java (hunk 1 above)
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java]: Hmm, a regular if would be more straight forward in test body :)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java [hunk 2]
```diff
@@ -50,6 +50,10 @@ public String getWikiName() {
     return this.wikiNameString;
   }
 
+  public String getDomainSufix() {
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java]: I think the lombok getter will do it for you. No need to add this getter

File: src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java [hunk 3]
```diff
@@ -25,8 +25,14 @@ public void createNewWikiLangTC001(String lang) {
     WikiBasePageObject base = new WikiBasePageObject();
     CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
     cnw1.selectLanguage(lang);
-    String expectedDomainPrefix = "en".equals(lang) ? "" : String.format("%s.", lang);
-    assertEquals(cnw1.getDomainPrefix().getText(), expectedDomainPrefix);
+    String expectedDomainSufix;
+    if (!lang.equals("en")) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java]: MAJOR Move the "en" string literal on the left side of this string comparison. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)

File: src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java [hunk 4]
```diff
@@ -25,8 +25,14 @@ public void createNewWikiLangTC001(String lang) {
     WikiBasePageObject base = new WikiBasePageObject();
     CreateNewWikiPageObjectStep1 cnw1 = base.openSpecialCreateNewWikiPage(wikiCorporateURL);
     cnw1.selectLanguage(lang);
-    String expectedDomainPrefix = "en".equals(lang) ? "" : String.format("%s.", lang);
-    assertEquals(cnw1.getDomainPrefix().getText(), expectedDomainPrefix);
+    String expectedDomainSufix;
+    if (lang.equals("en")) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLang.java]: MAJOR Move the "en" string literal on the left side of this string comparison. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)

**Pull request merged**